### DISCUSSION TITLE
Read Data Size Issue

### DISCUSSION THREAD

**Discussion by degnarraer** [pschatzmann/ESP32-A2DP#155] [Q&A] [answered]
read_data_stream data length is too big for my app, how can I set it.

Currently it is outputting 1024 samples (4096 bytes) every call. I am pushing that data to queues and that is not making my app very happy. I would like the call back to be called way more frequently, but with less data. Is there a way that I can set this somewhere?

**Comment by pschatzmann** [comment, marked as answer]
I am not aware of any functionality which would allow this. 
You could consider to combine the 2 channels into one and/or convert it from int16_t to int8_t.
